test/test-readability.js

```diff
@@ -47,11 +48,44 @@ var testPages = fs.readdirSync(testPageRoot).map(function(dir) {
   };
 });
 
+function suite(result, expectedContent, expectedMetadata) {
+  it("should return a result object", function() {
+    expect(result).to.include.keys("content", "title", "excerpt", "byline");
+  });
+
+  it("should extract expected content with jsdom", function() {
```

Comment:
Nit: This prints out `should extract expected content with jsdom` whether jsdom or JSDOMParser is used.